### DISCUSSION TITLE
July community call

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#2132] [Community Call]
### Asking questions/discussing things -
Start a new thread, and we can get responses on both the call and inside this github Discussion :)
You can drop an intro in the [introduction thread below](https://github.com/EddieHubCommunity/support/discussions/2132#discussioncomment-1094522) :)

**Comment by eddiejaoude** [comment]
Please introduce yourself 👇 and sharing social links

**Reply by naomi-lgbt** [reply]
Heya folks I'm nhcarrigan! You might know me as the one who built Becca, the discord bot that helps power our community.

**Reply by AbhayRaghuram** [reply]
Hello, I am Abhay. I am a code newbie and I am still learning Javascript.

**Reply by nitinsh06** [reply]
Hi guys, I'm **Nitin**, I use python for pretty much all stuffs, and getting better at c++ and js.

**Reply by ashwinexe** [reply]
Hello Everyone! I am Ashwin. I am an open source and web dev enthusiast. I just try to keep learning new tech.
Github: https://github.com/ashwinexe

**Reply by Vyvy-vi** [reply]
Hi, I'm Vyvy-vi. 
I might be the person with the potato profile pic, and the display name that doesn't make sense.

:) I like to help out, and I use Python, C. I also like making bots and have been very interested in APIs lately.

**Comment by eddiejaoude** [comment]
Q. Has Open Source helped you with your journey?

**Reply by SashoStoichkov** [reply]
YESSSSSS! 💯 

- Meeting new people
- Sharing knowledge
- Learning in public
- Hanging out with people with similar interests

**Reply by anonthedev** [reply]
Never really contributed to open source

**Reply by Vyvy-vi** [reply]
YES, very much so!

:)

**Reply by eddiejaoude** [reply]
@Anonymoususer70 not yet 😉

**Reply by anonthedev** [reply]
> @Anonymoususer70 not yet 😉

true

**Reply by avinashupadhya99** [reply]
I believe it has, I've met a lot of interesting folks through contributing to a few projects. Also learnt a lot of new tech through open source. There has been a couple of times where I was using a library and fortunately it was open source, so I could check the source code and implement the library better in my project.

**Reply by MitanshuShaBa** [reply]
I learnt about so many new things from the stargate project

**Reply by SaraJaoude** [reply]
To read about @nhcarrigan's journey into tech: (https://www.eddiejaoude.io/blog-july-2021-nicholas-carrigan)

**Reply by DevCassie** [reply]
Yes, open-source helped me with my journey. Not only for the skills but also for the connections. It's an amazing opportunity to create connections.

**Reply by nitinsh06** [reply]
Quite a lot, and, not that I contributed to a bunch of open-source projects but it feels good to be a part of an open-source community with so many brilliant people. 

I like visiting people's GitHub btw, there is always something new to learn 😉.

**Comment by Vyvy-vi** [comment]
Q) What's something interesting you learned or contributed to this month?

**Reply by eddiejaoude** [reply]
NextJS and NuxtJS hello world 🤓 - on live streams

**Reply by avinashupadhya99** [reply]
I started with Ruby this month. The best part was reading `Head first Ruby` and it was a really fun experience. It was my first time reading a Head First book and they are awesome. I also started with k8s but not much progress in either of them unfortunately.

**Reply by SashoStoichkov** [reply]
Learned NuxtJS from @eddiejaoude's stream ⬆️

**Reply by DevCassie** [reply]
I learned that creating/designing a portfolio for yourself is very hard 😂

**Reply by Vyvy-vi** [reply]
I picked up [FastAPI](https://fastapi.tiangolo.com/) and also built an application on [Uvicorn](https://www.uvicorn.org/), without any framework on top. I solved some Project Euler problems, which was fun.

**Reply by AbhayRaghuram** [reply]
I learned about what REST API is and how it works.
-Practicing DOM in javascript.
-Building mini projects.
-Started learning Python along with Javascript because of university.

**Reply by MitanshuShaBa** [reply]
I started learning golang, trying to wrap my head around how to write concurrent code well

**Reply by importhuman** [reply]
Contributed to CNCF buildpacks, learning cloud native fundamentals. Recently used ArgoCD for the first time!

**Reply by nitinsh06** [reply]
I learned [GraphQL API](https://graphql.org) this month, it's really efficient idea.

**Comment by eddiejaoude** [comment]
Q. Who is looking forward to EddieCon 0.1? http://eddiehub.org/eddiecon-01

**Reply by naomi-lgbt** [reply]
MEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Reply by avinashupadhya99** [reply]
Can't wait!

**Reply by Vyvy-vi** [reply]
MEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Reply by SaraJaoude** [reply]
Of course!

**Reply by RemcoHalman** [reply]
Looking Forward to it!

**Reply by naomi-lgbt** [reply]
![image](https://user-images.githubusercontent.com/63889819/127744872-db3d92db-add8-46fa-a8a6-3972c7141e55.png)

**Comment by avinashupadhya99** [comment]
Q. Any tips for planning and scheduling your day? I feel like I have trouble coming up with a schedule and sticking to it. I tried using notion to create a task board but that did not work. Any suggestions for tools to use? Thanks.

**Reply by eddiejaoude** [reply]
I use GitHub Project boards - it is amazing 🤓 

Some tips:
- break tasks down to as small as possible
- log everything
- columns: to do, in progress, blocked, done

**Reply by naomi-lgbt** [reply]
https://habitica.com/

![image](https://user-images.githubusercontent.com/63889819/127744951-7239bce7-b039-4db0-a4fc-52ced25beafa.png)

**Reply by naomi-lgbt** [reply]
![image](https://user-images.githubusercontent.com/63889819/127744956-b5a9032f-cef4-4859-8500-4fd2a7056f59.png)

**Reply by AbhayRaghuram** [reply]
-Trello
-Google Calender+Task+reminder 
I find google has worked best for me because you can subscribe to your own calender which emails you at 4am everyday with your schedule and tasks.

**Reply by Vyvy-vi** [reply]
- Google Calendars (with a ton of rainbow coloured event boxes)
- 😬  A big checklist notepad that lies on my table

**Reply by Vyvy-vi** [reply]
(mainly because I want to keep away from devices for like 6-7 hours, and I could still use my checklist when studying)

**Comment by MitanshuShaBa** [comment]
Any tips on a good automated API testing setup on Github Actions?

**Reply by eddiejaoude** [reply]
API repo with lots of automation https://github.com/EddieHubCommunity/api

**Comment by importhuman** [comment]
What are your thoughts on encouraging getting green squares on GitHub? For new folks it's definitely encouraging, but do you reckon it can also lead to missing the contribution itself for that visible square?

Editing for detail: I'm not saying small contributions are not valuable. Every contribution matters. I'm concerned that pushing for that square might take away from the process of contributing itself. I believe one should take the time they need to work without the pressure of getting a square on GitHub. (Hopefully I can sort out my mic to speak more in detail 😅 )

**Reply by AbhayRaghuram** [reply]
As a beginner, what I have been doing so far to be more active is this:
- 50 projects in 50 days by Traversey Media on Udemy. So my plan is push that completed project daily. 
- I also push my other learning projects on to a private repository.

**Reply by importhuman** [reply]
@AbhayRaghuram Yes that is what I'm saying is great. A daily goal is a great way to be regular. I'm of the opinion that this goal is what should be primary, not how many squares one has

**Reply by AbhayRaghuram** [reply]
> @importhuman  Yes that is what I'm saying is great. A daily goal is a great way to be regular. I'm of the opinion that this goal is what should be primary, not how many squares one has

Thats the way I approach it, I also had no idea about the green squares until after I started pushing more projects. 😅

**Reply by eddiejaoude** [reply]
I think green squares are fun, but should not be the goal, adding value should be the goal - it could be small or big, ideally small

**Reply by Vyvy-vi** [reply]
I think whenever we converse in chat about "green squares", it's mainly a reference to "did you add value today? did you make any contributions today?"

**Reply by Vyvy-vi** [reply]
The consistency reference refers to - "not making 1000s of contributions just one day, rather consistently working on projects"

**Reply by Vyvy-vi** [reply]
Even conversing in the github issue, or even helping anyone counts as "adding value"

**Comment by ashwinexe** [comment]
There are amazing communities which help everyone introduce to open source, promote free and great resources and knowledge by hosting events, talks with amazing folks, hackathons etc. but for a newly build community (with no experience), What can they do to get started?

**Reply by Vyvy-vi** [reply]
I think giving time to people involved in the community might be useful.
Helping out other people(primarily with dev stuff) can help you in onboarding people to github or new tech

**Reply by eddiejaoude** [reply]
the community needs to provide value, helping people is the best way to grow - but don't worry about the vanity metrics this will come with time
